sgv2-restapi/src/main/java/io/stargate/sgv2/restsvc/resources/Sgv2RowsResourceApi.java

```diff
@@ -37,6 +39,13 @@
     produces = MediaType.APPLICATION_JSON,
     consumes = MediaType.APPLICATION_JSON,
     tags = {"data"})
+@ApiImplicitParams({
+  @ApiImplicitParam(
+      name = "X-Cassandra-Token",
+      paramType = "header",
+      value = "The token returned from the authorization endpoint. Use this token in each request.",
+      required = true)
+})
 public interface Sgv2RowsResourceApi {
```

Comment:
Thankfully this works at the class level, we don't need to repeat it for every method. I've checked the generated Swagger docs.

Reply:
Nice!